### PR TITLE
Collection card styling

### PR DESCRIPTION
**Pull request by insomnious**
First pass at collection card styling. This features: 

- Text overflow handling of title, categories and summary text
- Success highlight of health regardless of value
- Basic icons added to get us started
- Overflow not handled for buttons and avatar areas

### CODE REVIEW

**Review by captainsandypants — APPROVED** [review]
Looks great.

File: src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/CollectionCards/CollectionCardStyles.axaml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                        <Style Selector="^ Image#TileImage">
+                            <Setter Property="Width" Value="144" />
+                            <Setter Property="Height" Value="180" />
+                        </Style>
+                    </Style>
+
+                    <!-- right column -->
+                    <Style Selector="^ panels|FlexPanel#TitleContainer">
+
+                        <!-- <Setter Property="Background" Value="DarkBlue" /> -->
```
Comment (Al12rs) [inline src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/CollectionCards/CollectionCardStyles.axaml:64]: Debating whether to keep these or not, they are actually pretty nice to quickly locate the selected element in the preview, so leaning towards keeping these

**Review by Al12rs — CHANGES_REQUESTED** [review]

File: src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/CollectionCards/CollectionCardStyles.axaml (hunk 1 above)
Comment (insomnious) [reply src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/CollectionCards/CollectionCardStyles.axaml:64]: I'd like to keep for now, we will be doing more passes to these styles as we are far from final

File: src/NexusMods.App.UI/Pages/LibraryPage/Collections/CollectionCardView.axaml [hunk 2]
```diff
@@ -11,64 +11,89 @@
...
+                        <Image x:Name="TileImage" />
+                    </Border>
+
+                    <panels:FlexPanel x:Name="TitleContainer">
+                        <TextBlock x:Name="TitleText" />
                         <Border x:Name="CategoryTextBorder">
-                            <TextBlock x:Name="CategoryText" Classes="BodySMNormal"/>
+                            <panels:FlexPanel x:Name="CategoryTextWrapper">
+                                <TextBlock x:Name="CategoryText" />
+                            </panels:FlexPanel>
```
[the remark this answers is not in the document]
Comment (insomnious) [reply src/NexusMods.App.UI/Pages/LibraryPage/Collections/CollectionCardView.axaml]: This one has been removed

**Pull request merged**